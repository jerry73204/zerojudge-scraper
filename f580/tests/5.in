12 56
6 -1
12 2
3 -1
2 -2
1 2
7 5
5 -1
5 11
1 4
6 -2
8 7
8 7
10 4
6 10
3 12
12 -2
2 11
3 4
4 3
12 5
11 6
8 -2
8 11
6 10
2 5
12 -1
5 -2
6 9
10 4
10 -1
9 -2
12 8
4 -2
8 4
6 8
8 11
7 10
3 10
2 12
10 -1
6 12
7 -1
10 4
8 9
6 -1
10 3
3 -1
3 11
12 7
10 3
9 7
11 5
9 7
8 10
9 10
5 -2
