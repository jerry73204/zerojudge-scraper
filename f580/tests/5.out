1 5 1 3 2 6 5 1 3 5 3 6